docs/troubleshooting.md

```diff
@@ -47,3 +47,120 @@ dataprotectionapplication-1   Available   33s              8d    true
 ```
 
 Otherwise, user should manually remove all backup resources `backups.velero.io`, `restores.velero.io` and `deletebackuprequests.velero.io`. Then add `lca.openshift.io/manual-cleanup-done` annotation to IBU CR.
+
+## LVMS volume contents not restored
+
+When LVMS is used to provide dynamic PV storage, it may not restore the PV contents when it is not configured correctly.
+Below, we provide some common symptoms and solutions in order to aid the user while troubleshooting such cases.
+
+### Missing lvms-related fields in Backup CR
+
+#### Symptom (missing backups fields)
+
+```shell
+-> oc describe pod <sample-app-name>
+
+... TRUNCATED ...
+
+Events:
+  Type     Reason            Age                From               Message
+  ----     ------            ----               ----               -------
+  Warning  FailedScheduling  58s (x2 over 66s)  default-scheduler  0/1 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/1 nodes are available: 1 Preemption is not helpful for scheduling..
+  Normal   Scheduled         56s                default-scheduler  Successfully assigned default/db-777b7ff898-g8gg6 to sno1.inbound.bos2.lab
+  Warning  FailedMount       24s (x7 over 55s)  kubelet            MountVolume.SetUp failed for volume "pvc-e8bf06ad-62a2-418b-ab29-c09e64d5550e" : rpc error: code = Unknown desc = VolumeID is not found
+```
+
+#### Resolution (missing backups fields)
+
+The reason for this issue is that the application has restored its PVC and PV manifests correctly, but the
+`logicalvolume` associated to this PV hasn't been restored properly after pivot.
+Steps for its resolution were described in [Section 1.1 - Backup resource when LVMS is used](backuprestore-with-oadp.md#11-backup-resource-when-lvms-is-used),
+and basically relates to also include `logicalvolumes.topolvm.io` in the application's Backup CR (see sample below).
+
+```yaml
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: small-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  includedClusterScopedResources:
+  - persistentVolumes              # <- required field
```

Comment:
I assume that all PVs will be backed up. If there are PVs created by both LSO and LVMS, will the PVs created by LSO be affected after restore in the post-pivot?

Reply:
We need to update the documentation, only LSO or LVMS can be used, not both

Reply:
should we need to drop a note for that in this PR, or will it be included later on (in the official docs, for instance)?

Reply:
We should do both

Reply:
Ok, just dropped [a note for it here](https://github.com/openshift-kni/lifecycle-agent/pull/448/commits/3e5e750549f3ca5cef80217ceb1938f75961a720#diff-d08037f5ce2c67f942ba8bf93d5d9c83711172f9d289737d2a44407044d1a4f8R447-R449)

/cc @amolnar-rh   (keeping you in the loop - for when adding LVMS support to official docs)